package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "last-train-to-nodeville",
+  "version": "1.0.0",
+  "description": "In the initial commit to this project, you have been provided with files in an initial commit that contain a JavaScript module and some associated tests. Your task is to use NPM to setup the necessary project dependencies to make it possible for the command `npm test` to successfully run the test suite.",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
```

Comment:
Take a look at Slide 52. In there we show that you can pass a pattern to mocha for it to find our tests files. We will always want to specify rather than letting it figure that out on its own so we don't run into issues. In this case we would simply pass it the file name since we only have one test file.

`"test": "./node_modules/.bin/mocha 'tests.js'"`